openkfw/TruBudget
# 1832 filesize

### Checklist




### Description


### How to test

0. storage service has to be enabled
1. login as any user, e.g. mstein
2. try to create several workflow items: with attached file, with multiple files, without file, etc...
3. everything should work as before (downloaded file is the same as uploaded, ...)




Closes #1832
